storage/diskmetricstore.go

```diff
@@ -32,9 +34,15 @@ import (
 )
 
 const (
-	writeQueueCapacity = 1000
+	pushMetricName       = "push_time_seconds"
+	pushMetricHelp       = "Last Unix time when this group was changed in the Pushgateway."
```

Comment:
Yeah, I guess we have to change that. Will create inconsistent help string warnings, but they will go away eventually…

Reply:
My thought was that the old help string is fine because a failed push will not really _change_ the group. It will just leave it unchanged (with the exception of the failure timestamp metric). But I guess, a clearer wording is better.